contracts/governance/extensions/GovernorVotesQuorumFractional.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "./GovernorVotes.sol";
+
+/**
+ * @dev Extension of {Governor} for voting weight extraction from an {ERC20Votes} token and a quorum expressed as a
+ * fraction of the total supply.
+ *
+ * _Available since v4.3._
+ */
+abstract contract GovernorVotesQuorumFractional is GovernorVotes {
+    uint256 private _quorumRatio;
+
+    event QuorumRatioUpdated(uint256 oldQuorumRatio, uint256 newQuorumRatio);
+
+    constructor(uint256 quorumRatioValue) {
+        _updateQuorumRatio(quorumRatioValue);
+    }
+
+    function quorumRatio() public view virtual returns (uint256) {
+        return _quorumRatio;
+    }
+
+    function quorumRatioMax() public view virtual returns (uint256) {
+        return 100;
+    }
+
+    function quorum(uint256 blockNumber) public view virtual override returns (uint256) {
+        return (token.getPastTotalSupply(blockNumber) * quorumRatio()) / quorumRatioMax();
+    }
+
+    function updateQuorumRatio(uint256 newQuorumRatio) external virtual onlyGovernance() {
```

Comment:
Does Prettier have an opinion about this? I prefer no parentheses.

```suggestion
    function updateQuorumRatio(uint256 newQuorumRatio) external virtual onlyGovernance {
```

Reply:
I prefer parentheses, and I don't think prettier cares.